lib/std/Thread/WaitGroup.zig

```diff
@@ -33,20 +36,22 @@ pub fn wait(self: *WaitGroup) void {
     }
 }
 
+/// Resets the wait group to its initial state for reuse.
 pub fn reset(self: *WaitGroup) void {
     self.state.store(0, .monotonic);
     self.event.reset();
 }
 
+/// Checks if all tasks have completed.
```

Comment:
```suggestion
/// Returns `true` if the wait group counter is zero, `false` otherwise. Thread-safe.
/// Depending on the behavior of other threads, it may be a race condition to rely on this value.
```